### ISSUE TITLE
Nuclear fallout to long

### ISSUE THREAD

**Issue by m3nda** [skyarkhangel/Hardcore-SK#1550]
tldr
What about: Nuclear fallout after nuke hit looks like fixed on 100 days.

Problem: So long time of nuclear fallout makes game boring.

Proposal solve: Cut it to 25% of original duration time.
/tldr 

After getting hit by nuclear warhead we are getting nuclear fallout. This fallout duration looks like being fixed on 100 days. Problem in it is that it makes game seriously boring as for so long time pawns are almost closed in base (also looks like incidents are more rare, but i'm not sure about this).
My proposal is to cut it to 25% of original duration time, if it gonna make your colony killed, it will make it in this period of time.

**Comment by erm3nda** [comment]
Gime me your account :-/
